store/sqlstore/utils.go

```diff
@@ -52,3 +53,26 @@ func finalizeTransaction(transaction *gorp.Transaction) {
 		mlog.Error("Failed to rollback transaction", mlog.Err(err))
 	}
 }
+
+// removeNonAlphaNumericTerms removes all words that only contains non-alphanumeric chars from given line
+func removeNonAlphaNumericTerms(line string) string {
+	words := strings.Split(line, " ")
+	filteredResult := make([]string, 0, len(words))
+
+	for _, w := range words {
+		if containsAlphaNumericChar(w) {
+			filteredResult = append(filteredResult, strings.TrimSpace(w))
+		}
+	}
+	return strings.Join(filteredResult, " ")
+}
+
+// isAlphaNumericWord returns true in case any letter or digit is present, false otherwise
+func containsAlphaNumericChar(s string) bool {
+	for _, r := range s {
```

Comment:
Probably you need to work with runes. We support languages like Chinese or Japanese

Reply:
> Range loops
> ...
> We've seen what happens with a regular for loop. A for range loop, by contrast, decodes one UTF-8-encoded rune on each iteration. Each time around the loop, the index of the loop is the starting position of the current rune, measured in bytes, and the code point is its value. Here's an example using yet another handy Printf format, %#U, which shows the code point's Unicode value and its printed representation:

As [doc](https://blog.golang.org/strings) says, `for` loop with `range over string` handles the rune

Reply:
Cool. Just wanted to be sure. I didn't work with runes too much :D